skbio/io/format/blast6.py

```diff
@@ -0,0 +1,287 @@
+"""
+BLAST+6 format (:mod:`skbio.io.format.blast6`)
+=========================================================================
+
+.. currentmodule:: skbio.io.format.blast6
+
+The BLAST+6 format (``blast6``) stores strings, integers, and floats relating
+to the similarities between multiple DNA sequences. These values are stored in
+a simple tabular format with no labels on individual columns. The values are
+separated by the tab character.
+
+An example BLAST+6-formatted file comparing two protein sequences, taken
+from [1]_::
+
+    moaC    gi|15800534|ref|NP_286546.1|    100.00  161 0   0   1   161 1
```

Comment:
To emphasize the tab characters, can you insert `<tab>` between the values? These lines also need a line continuation (each line is currently rendering as two lines).